src/generators/schema/breadcrumb.php

```diff
@@ -79,15 +83,30 @@ public function generate( Meta_Tags_Context $context ) {
 				$broken = true;
 				break;
 			}
-			$list_elements[] = $this->add_breadcrumb( $index, $breadcrumb, $context );
+
+			if ( $index === ( $nr_of_breadcrumbs - 1 ) ) {
+				/*
+				 * Fall back to the current URL and/or title,
+				 * but only for the last breadcrumb, which corresponds to the
+				 * current webpage.
+				 */
+				if ( empty( $breadcrumb['url'] ) ) {
```

Comment:
This check is only for the last value of breadcrumbs. Maybe we should do this check outside the foreach.